manifest/package.xml

```diff
@@ -1,12 +1,17 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>*</members>
-        <name>ApexClass</name>
-    </types>
-    <types>
-        <members>*</members>
-        <name>LightningComponentBundle</name>
-    </types>
-    <version>52.0</version>
-</Package>
+	<types>
+		<members>*</members>
+		<name>ApexClass</name>
+	</types>
+	<types>
+		<members>recordDeletedSuccessMessage</members>
+		<members>recordUpdatedSuccessMessage</members>
```

Comment:
@anupkage let's include all *, because we are include all. because we have to change for all components, let keep *